hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -49,7 +49,48 @@
             '''
       milestone: V2
       tests: ["pattgen_perf"]
-    }   
+    }
+    {
+      name: cnt_rollover
+      desc: '''
+            Checking ip operation with random counter values
+
+            Stimulus:
+              - Program the pre-divider and size registers to unconstraint random values
+              - Program the clk_cnt, bit_cnt and rep_cnt to values less than but close to
+                predivider and size registers, so that counting would take a reasonable
+                number of clock cycles
+              - include programming for corner cases
+                repeat programming a random number of times
+              - Start and stop channels quickly
+              - Clear interrupts quickly
+            Checking:
```

Comment:
Suggest adding a newline before this. That will make the 'Checking' part appear as a new paragraph in the generated DV document.

Reply:
Added a newline before "Checking"